azure-common/azure/profiles/__init__.py

```diff
@@ -149,6 +149,53 @@ class KnownProfiles(Enum):
         },
         "2018-03-01-hybrid"
     )
+    v2019_03_01_hybrid = ProfileDefinition(
+        {
+            "azure.keyvault.KeyVaultClient": {
+                None: "2016-10-01"
+            },
+            "azure.multiapi.storage": {
```

Comment:
note that this is a new addition that we do not have it in the previous profile sdk.

Reply:
@lmazuel Laurent, should we remove data-plane storage & cosmosdb from this profile?

Reply:
We can keep them, but they are not used at all, since there is no support for reading this profile for now.

I would be specific, and remove them until this is truly supported.